app/views/reports/regions/_child_data_tables.html.erb

```diff
@@ -91,11 +110,32 @@
           <%= @region.name %>
         </td>
         <td class="ta-right">
-          <%= number_with_delimiter(@data.dig(:cumulative_registrations, @period)) %>
+          <%= number_with_delimiter(@data.dig(:registrations, @period)) %>
         </td>
         <td class="ta-right">
-          <%= number_with_delimiter(@data.dig(:registrations, @period)) %>
+          <%= number_with_delimiter(@data.dig(:cumulative_registrations, @period)) %>
         </td>
+        <%= can_manage_region = accessible_region?(@region, :manage) %>
```

Comment:
The org report breaks while checking `accessible_region?` with `undefined method 'accessible_organization?'`.

Reply:
Fixed the RegionAccess model: https://github.com/simpledotorg/simple-server/pull/4794/files#diff-e69f0b691aba3f662ff395367fbeccc1d8965d336cfeeb1fce58cc98b5c1a179R13

Reply:
Because we don't want to display this for organisations? Makes sense.